packages/client/hmi-client/src/components/dataset/tera-dataset.vue

```diff
@@ -217,6 +283,22 @@ const card = computed(() => {
 const description = computed(() => dataset.value?.description?.concat('\n', card.value?.DESCRIPTION ?? '') ?? '');
 const author = computed(() => card.value?.AUTHOR_NAME ?? '');
 
+function updateColumn(index: number, key: string, value: any) {
+	if (!transientDataset.value?.columns?.[index]) return;
+	if (key === 'unit' || key === 'name') {
+		transientDataset.value.columns[index].metadata[key] = value;
```

Comment:
metadata is optional
If I have an unenriched dataset and the metadata does not exist for my column then this will fail.